nested-spire/README.md

```diff
@@ -0,0 +1,226 @@
+The other point to highlight is the `-downstream` option. This option, when set, indicates that the entry describes a downstream SPIRE Server.
+
+## Run the Scenario
+
+Use the `set-env.sh` script to run all the services that make up the scenario. The script starts the `root`, `nestedA`, and `nestedB` services with the configuration options described earlier.
+
+Ensure that the current working directory is `.../spire-tutorials/nested-spire` and run:
+
+```console
+    bash scripts/set-env.sh
+```
+
+Once the script is completed, in another terminal run the following command to review the logs from all the services:
+
+```console
+    docker-compose logs -f -t
+```
+
+
+# Part 2: Test the Deployments
+
+Now that the SPIRE deployments are ready, let's test the scenario that we've configured.
+
+## Create Workload Registration Entries
+
+To test the scenario we create two workload registration entries, one entry for each nested SPIRE Server (`nestedA` and `nestedB`). The goal of the test is to demonstrate that SVIDs created in a nested configuration are valid in the entire trust domain, not only in the scope of the SPIRE Server that originated the SVID. The following commands demonstrate the command line options we'll use to create the two workload registration entries, but you can run these commands using the `create-workload-registration-entries.sh` script shown a few lines below.
+
+```console
+   # Workload for nestedA deployment
+   docker-compose exec -T nestedA-server \
+       /opt/spire/bin/spire-server entry create \
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint nestedA/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedA/workload" \
+       -selector "unix:uid:1001" \
+       -ttl 0
+
+   # Workload for nestedB deployment
+   docker-compose exec -T nestedB-server \
+       /opt/spire/bin/spire-server entry create \
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint nestedB/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedB/workload" \
+       -selector "unix:uid:1001" \
+       -ttl 0
+```
+
+The examples use the `$(fingerprint nestedA/agent/agent.crt.pem)` form again to show that the `-parentID` flag specifies the SPIFFE ID of the nested SPIRE Agent. The TTL flag value of zero indicates that the default value of 3600 seconds will be used. Finally, in both cases the unix selector assigns the SPIFFE ID to any process with a uid of 1001.
+
+Use the following Bash script to create the registration entries using the options just described:
+
+```console
+   bash scripts/create-workload-registration-entries.sh
+```
+
+## Run the Test
+
+Once both workload registration entries are propagated, let's test that SVIDs created in a nested configuration are valid in the entire trust domain, not only in the scope of the SPIRE Server that originated the SVID.
```

Comment:
how can they verify that workload registration entries are propagated?

Reply:
The output of the bash script shows that the entries are propagated, there is no extra step here for the reader.

Reply:
maybe you can add a simple way to verify it on readme? for people that dont want to run bash, and they can verify it esasly?